mkdocs.yml

```diff
@@ -77,7 +77,9 @@ markdown_extensions:
   - pymdownx.inlinehilite
   - pymdownx.snippets
   - pymdownx.superfences
-
+extra_javascript:
```

Comment:
If there's a specific issue or SO question that we could add here as a reference on why this is needed, it would be helpful for future us.

Reply:
I just followed the official docs, I added a comment with a link.